users/fml128/config.h

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2021 Merlin Jehli <merlin@jeh.li> @fml128
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#undef MANUFACTURER
+#undef PRODUCT
+#undef DESCRIPTION
+
+
+#define MANUFACTURER        "FML128.ch"
+#define PRODUCT             "FMK128"
+#define DESCRIPTION         "FMK128"
```

Comment:
```suggestion
#define MANUFACTURER        FML128.ch
#define PRODUCT             FMK128
```

Reply:
This change resulted in an error. I have removed the description, but the " needs to be there.